public/index.html

```diff
@@ -8,5 +8,6 @@
 </head>
 <body>
   <div id="radioPlayer"></div>
+  <div id="portal"></div>
```

Comment:
You shouldn't need to add #portal div manually. Your Modal compoennt should be able to create a container for itself dynamically every time it's invoked. For example, you could dynamically create an HTML element in `useEffect` hook callback.

```js
const portal = document.createElement('div');
```

And then append that element to body:

```js
document.body.appendChild(portal);
```

then you can pass it to your `createPortal` function call:

```js
return ReactDOM.createPortal(
    ...,
    portal
);
```

Remember that you'll have to remove that element when your modal is unmounted (in other words: disappears from the page completely). In order to do that, you should return a function from `useEffect` callback, and in that function remove the portal container, like so: `portal.parentNode.removeChild(portal)`